java-client/src/main/java/org/opensearch/client/json/JsonpUtils.java

```diff
@@ -68,7 +68,7 @@ static JsonProvider provider() {
         return JsonProvider.provider();
     }
 
-    static final JsonpMapper DEFAULT_JSONP_MAPPER = new JsonpMapperBase() {
+    public static final JsonpMapper DEFAULT_JSONP_MAPPER = new JsonpMapperBase() {
```

Comment:
Sorry, I missed that, please keep it package private:

```suggestion
    static final JsonpMapper DEFAULT_JSONP_MAPPER = new JsonpMapperBase() {
    
```